sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/InvocationVisitor.cs

```diff
@@ -104,13 +109,32 @@ private ProgramState HandleStringNullCheckMethod()
                 .PopValue(out var arg1)
                 .PopValue();
 
-            // Handle string.IsNullOrEmpty(arg1) as if it was arg1 == null
+            // Handle string.IsNullOrSpace(arg1) as if it was arg1 == null

```

Comment:
IsNullOrSpace or IsNullOrWhiteSpace? 

what does this mean? I see no IsNullOrWhiteSpace here - should this comment be here or elsewhere?